src/main/telemetry/crsf.c

```diff
@@ -413,13 +414,13 @@ void crsfFrameFlightMode(sbuf_t *dst)
     if (!ARMING_FLAG(ARMED) && !FLIGHT_MODE(FAILSAFE_MODE)) {
         // * - ready to arm
         // ! - arming disabled
-        // ? - GPS fix not available
```

Comment:
Should say “GPS Home Fix not available”.
To distinguish between a 3D fix and a Home fix.

Reply:
Seems your are reviewing a previous version.

Has been updated to

// ? - GPS rescue disabled (when allowArmingWithoutFix is enabled)